examples/eks-quickstart-app-dev.yaml

```diff
@@ -0,0 +1,27 @@
+# An example of ClusterConfig object compatible with the "app-dev" quickstart
+# profile. See also: https://github.com/weaveworks/eks-quickstart-app-dev
+---
+apiVersion: eksctl.io/v1alpha5
+kind: ClusterConfig
+
+metadata:
+  name: cluster-1
+  region: eu-north-1
+
+nodeGroups:
+  - name: ng-1
+    instanceType: m5.large
+    minSize: 1
+    maxSize: 2
+    iam:
+      withAddonPolicies:
+        albIngress: true
+        autoScaler: true
+        cloudWatch: true
+
+cloudWatch:
+    clusterLogging:
+        # enable specific types of cluster control plane logs
+        enableTypes: ["audit", "authenticator", "controllerManager"]
```

Comment:
It isn't mandatory to have control plane logging to CloudWatch enabled in order for the CloudWatch agent to work.

Reply:
If removing `cloudWatch.clusterLogging`,

- `/aws/containerinsights/${CLUSTER_NAME}/application` is still there in CloudWatch
- `/aws/containerinsights/${CLUSTER_NAME}/dataplane` is still there in CloudWatch
- `/aws/containerinsights/${CLUSTER_NAME}/host` is still there in CloudWatch
- `/aws/containerinsights/${CLUSTER_NAME}/performance` is still there in CloudWatch
- `/aws/eks/${CLUSTER_NAME}/cluster` **is no longer there in CloudWatch**

hence we still get container-level logs.